### PR TITLE
update caption in ap slide

### PR DESCRIPTION
**Pull request by APBEI**
### Caption

this pull request changes the caption for AP slide. This relate to #4 
@PoloEIB does this look nice ?

### CODE REVIEW

**Comment by PoloEIB** [comment]
No it doesn't

File: _posts/0000-01-04-APBEI.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![kimonotocat](https://octodex.github.com/images/kimonotocat.png)
 {: .center}
 
-Caption here
+So good
```
Comment (APBEI) [inline _posts/0000-01-04-APBEI.md]: :octocat:

**Review by APBEI — COMMENTED** [review]
great

File: _posts/0000-01-04-APBEI.md (hunk 1 above)
Comment (ImSo06) [inline _posts/0000-01-04-APBEI.md]: 👍

**Review by ImSo06 — APPROVED** [review]

File: _posts/0000-01-04-APBEI.md (hunk 1 above)
Comment (alueib) [reply _posts/0000-01-04-APBEI.md]: 👍 👍

**Review by alueib — APPROVED** [review]

**Review by PoloEIB — APPROVED** [review]
Nice

**Pull request merged**